test/Microsoft.TemplateEngine.Cli.UnitTests/CliMocks/MockNewCommandInput.cs

```diff
@@ -209,5 +306,28 @@ public IReadOnlyList<string> VariantsForCanonical(string canonical)
         {
             throw new NotImplementedException();
         }
+
+        #region IXunitSerializable implementation
+        public void Deserialize(IXunitSerializationInfo info)
+        {
+            TemplateName = info.GetValue<string>("command_templateName");
+            _commandOptions = JsonConvert.DeserializeObject<Dictionary<string, string>>(info.GetValue<string>("command_options"));
+            _templateOptions = JsonConvert.DeserializeObject<Dictionary<string, string>>(info.GetValue<string>("command_templateOptions"));
+        }
+        public void Serialize(IXunitSerializationInfo info)
+        {
+            info.AddValue("command_templateName", TemplateName, typeof(string));
+            info.AddValue("command_options", JsonConvert.SerializeObject(_commandOptions), typeof(string));
+            info.AddValue("command_templateOptions", JsonConvert.SerializeObject(_templateOptions), typeof(string));
+        }
+
+        public override string ToString()
```

Comment:
minor: unless this string output is very necessary, consider simplifying it, specially if called often.  Why?  strings :(

Reply:
This output will be shown in data-driven tests row later own. I would prefer better output for tests.